index.html

```diff
@@ -65,7 +65,7 @@
             seed: seedPhrase,
             revocationOpts: {
                 type: CredentialStatusType.Iden3ReverseSparseMerkleTreeProof,
-                baseUrl: 'http://rhs.com/node'
+                id: 'http://rhs.com/node'
```

Comment:
If you pay attention to our credentials, we have an `id` field on the `credentialStatus` field. In the past, we only used ids that have http(s) URL, but now it can be `did`:
```
"credentialStatus": {
        "id": "https://dev.polygonid.me/api/v1/identities/did%3Apolygonid%3Apolygon%3Amumbai%3A2qNLo1aEbiLeJ9awYyGHRZx6Bszhqt9qVAZ88qcRzn/claims/revocation/status/3043138372",
        "revocationNonce": 3043138372,
        "type": "SparseMerkleTreeProof"
    },
```
I think `id` is correct name.

Reply:
Also, if we take into account the w3c documentation, it says that each resource must have an identifier. In our case, `credentialstatus` is a resource

Reply:
id is not right when you use it in `options`. Because later it can be concat with other params. e.g. `revocationNonce` in SparseMerkleProof

options have nothing with credentialStatus spec.
also we can shouldn't break interface.
I'm not sure how revocationOpts will be used with onchain status for now.

Reply:
@vmidyllic what do you think about it?

Reply:
We decided to use `id` instead of `baseUrl`